wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/provisioncredential/EvidenceRequest.kt

```diff
@@ -143,6 +165,161 @@ fun EvidenceRequestMessageView(
     }
 }
 
+/**
+ * Show a generic "detailed message" screen whose [detailedMessage.messageType] is not handled.
+ * Composes a screen containing the "info" icon, message title, message text and the accept button.
+ */
+@Composable
+fun EvidenceRequestGenericDetailedMessageScreen(
+    detailedMessage: EvidenceRequestDetailedMessage,
+    provisioningViewModel: ProvisioningViewModel,
+    walletServerProvider: WalletServerProvider,
+    documentStore: DocumentStore
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(horizontal = 10.dp)
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 100.dp),
+            horizontalArrangement = Arrangement.Center
+        ) {
+            Icon(
+                painter = painterResource(id = android.R.drawable.ic_dialog_info),
+                modifier = Modifier.size(80.dp),
+                contentDescription = stringResource(
+                    R.string.accessibility_artwork_for,
+                    detailedMessage.messageTitle
+                )
+            )
+        }
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.Center
+        ) {
+            Text(
+                text = detailedMessage.messageTitle,
+                modifier = Modifier.padding(16.dp),
+                textAlign = TextAlign.Left,
+                fontWeight = FontWeight.Bold,
+                style = MaterialTheme.typography.headlineLarge
+            )
+        }
+        Row(
+            modifier = Modifier.fillMaxWidth().padding(bottom = 200.dp),
+            horizontalArrangement = Arrangement.Absolute.Left
+        ) {
+            Text(
+                modifier = Modifier.padding(start = 16.dp, end = 16.dp),
+                text = detailedMessage.message,
+                style = MaterialTheme.typography.bodyLarge
+            )
+        }
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.End
+        ) {
+            Button(
+                modifier = Modifier.padding(8.dp),
+                onClick = {
+                    provisioningViewModel.provideEvidence(
+                        evidence = EvidenceResponseMessage(true),
+                        walletServerProvider = walletServerProvider,
+                        documentStore = documentStore
+                    )
+                }) {
+                Text(detailedMessage.acceptButtonText)
+            }
+        }
+    }
+}
+
+/**
+ * Show the "Document Scanning Complete" screen after completing all steps of for evidence gathering
+ * of the "Provisioning a Personal ID" flow.
```

Comment:
Is there anything in this screen that's specific to provisioning a PID? I'd drop this last part of the docstring, since this could potentially be reused for other completion flows.